lib/remote/httpserverconnection.hpp

```diff
@@ -31,15 +31,15 @@ class HttpServerConnection final : public Object
 	void Disconnect();
 	void StartStreaming();
 
-	bool Disconnected();
+	bool Disconnected() const;
 
 private:
 	ApiUser::Ptr m_ApiUser;
 	Shared<AsioTlsStream>::Ptr m_Stream;
 	double m_Seen;
 	String m_PeerAddress;
 	boost::asio::io_context::strand m_IoStrand;
-	bool m_ShuttingDown;
+	std::atomic<bool> m_ShuttingDown;
```

Comment:
Due to how the (not changed) constructor operates, I'd like to see our Atomic<> here (that's its purpose).

Reply:
I'm not entirely sure what you're referring to, but we don't need an atomic constructor here. Since it's not a shared memory, why do you want to use an atomic constructor here?